package-lock.json

```diff
@@ -21531,10 +21797,68 @@
       "resolved": "https://registry.npmjs.org/style-search/-/style-search-0.1.0.tgz",
       "integrity": "sha1-eVjHk+R+MuB9K1yv5cC/jhLneQI="
     },
+    "stylelint": {
```

Comment:
Hope we won't have problems, that this `stylelint` will be always behind. “Thanks”, npm 7.

Reply:
Where's this entry coming from, is it the jest-preset? We can set the peer to 14 when it's out.

Reply:
Yes, peerDependency of jest-preset.

Reply:
Shall we remove the peer dependency? It's odd having stylelint installed in stylelint automatically.

Reply:
I can do it as a patch release of the preset. We probably don't want an old version of stylelint (and postcss) hanging around as we migrate to PostCSS 8 it adds a bit of an unknown variable into the mix.

Reply:
I don't think anyone uses jest-preset to test stylelint plugin without stylelint installed. So it's probably safe to remove peer dependency.